### PR TITLE
Get Started - add get started layout

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### What change does this PR introduce?

Add get started layout.

### Why was this change needed?

In order to play all the sections on one page.

### CODE REVIEW

File: apps/web/src/App.tsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ import { Trigger } from './pages/quick-start/steps/Trigger';
 import { RequiredAuth } from './components/layout/RequiredAuth';
 import { TemplateFormProvider } from './pages/templates/components/TemplateFormProvider';
 import { TemplateEditorProvider } from './pages/templates/components/TemplateEditorProvider';
+import { GetStarted } from './pages/quick-start/steps/GetStarted';
+import { DigestPreview } from './pages/quick-start/steps/DigestPreview';
```
Comment (LetItRock) [inline apps/web/src/App.tsx:43]: these files don't exist in this PR, so it's not possible to check how it works :( 
Did you miss it?

File: apps/web/src/constants/routes.enum.ts [hunk 2]
```diff
@@ -13,6 +13,8 @@ export enum ROUTES {
   TEMPLATES_EDIT_TEMPLATEID = '/templates/edit/:templateId',
   TEMPLATES = '/templates',
   QUICKSTART = '/quickstart',
+  GET_STARTED = '/get-started',
+  DIGEST_PREVIEW = '/get-started/preview',
```
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply apps/web/src/constants/routes.enum.ts:17]: In general, I would love to go over the routes together :)

File: apps/web/src/constants/routes.enum.ts [hunk 3]
```diff
@@ -13,6 +13,8 @@ export enum ROUTES {
   TEMPLATES_EDIT_TEMPLATEID = '/templates/edit/:templateId',
   TEMPLATES = '/templates',
   QUICKSTART = '/quickstart',
```
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply apps/web/src/constants/routes.enum.ts:15]: Same here would love to go through the routes, in this case, we still have the in-app route (this one is the old initial page) so maybe we could update it.

File: apps/web/src/App.tsx (hunk 1 above)
Comment (djabarovgeorge) [reply apps/web/src/App.tsx:43]: I committed them by mistake on this PR I added them on https://github.com/novuhq/novu/pull/3006

**Review by LetItRock — APPROVED** [review]

**Pull request merged**